tests/test_coco_and_via_conversions.py

```diff
@@ -0,0 +1,342 @@
+import datetime
+import os
+from pathlib import Path
+
+import pytest
+
+from bboxes_labelling.annotations_utils import (
+    combine_multiple_via_jsons,
+    convert_via_json_to_coco,
+    read_json_file,
+)
+
+
+@pytest.fixture()
+def via_json_1() -> str:
+    """Return path to a sample VIA JSON file.
+
+    Returns
+    -------
+    str
+        path to a sample VIA JSON file 1.
+    """
+    # Return path to sample VIA (Visual Image Annotator) JSON file 1
+    return str(Path("tests/data/COCO_VIA_JSONS/VIA_JSON_1.json").resolve())
+
+
+@pytest.fixture()
+def via_json_2() -> str:
+    """Return path to a sample VIA JSON file.
+
+    Returns
+    -------
+    str
+        path to a sample VIA JSON file 2.
+    """
+    # Return path to sample VIA JSON file 2
+    return str(Path("tests/data/COCO_VIA_JSONS/VIA_JSON_2.json").resolve())
```

Comment:
Same here

Reply:
yes, good catch! I changed it so that it is relative to the path of the test...py file now, so that it works regardless of the directory you are running pytest from